mmdet/datasets/transforms.py

```diff
@@ -10,7 +10,6 @@
 
 class ImageTransform(object):
     """Preprocess an image.
-
```

Comment:
The blank line between the summary and detailed description is better to be kept.

Reply:
I agree, but the blank line is not present in any of the classes in the same file. Please let me know the changes that I should make. Thanks!

Reply:
For the docstrings that contains a summary and detailed description, usually we add a blank line between them. You can just recover the changes in such docstrings.

Reply:
Made those changes. I hope it's okay now. Please let me know. 

Also, does `mmdetection` accept packages written with PIL? I haven't seen any so far. So I wanted to confirm.

Reply:
Currently it works with numpy arrays. If you would like to play with PIL images, a wrapper may be needed.
```python
x = to_pil_img(x)
# some operations here
x = to_numpy_array(x)
```

Reply:
Seems that there are new linting errors.

Reply:
I recently wrote a package for Auto augmentation, but that makes use of PIL for certain transformations. Please let me know if I can contribute it with PIL or should I write it with cv2.

Reply:
I recommend using cv2, which is usually more efficient and has better compatibility with the current pipeline.

Reply:
Okay, I'll work on it and get back. Thanks for the response!

Reply:
> Okay, I'll work on it and get back. Thanks for the response!

@dhananjaisharma10 Hello, have you complete the Auto augmentation for mmdetection? I need to use it, but I don't know how to write it for mmdetection.